plugin/src/functionalTest/groovy/com/here/gradle/plugins/jobdsl/tasks/GenerateXmlTest.groovy

```diff
@@ -68,6 +68,21 @@ class GenerateXmlTest extends AbstractTaskTest {
         XMLUnit.compareXML(expectedText, actualText).identical()
     }
 
+    def 'folder is generated incorrectly'() {
```

Comment:
The name is not so telling, what about 'failOnMissingPlugin fails the task if a required plugin is missing'?

Reply:
Sounds great, I applied the change.